### PR TITLE
Add fr.free.hxc2001.HxCFloppyEmulator

### PR DESCRIPTION
**Pull request by FakeShemp**
http://hxc2001.free.fr/floppy_drive_emulator/

### CODE REVIEW

**Comment by FakeShemp** [comment]
bot, build fr.free.hxc2001.HxCFloppyEmulator

**Comment by FakeShemp** [comment]
bot, build fr.free.hxc2001.HxCFloppyEmulator

**Comment by barthalion** [comment]
It would be nice to contact upstream, but project's page doesn't even load for me.

File: fr.free.hxc2001.HxCFloppyEmulator.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        path: fltk-fix-lib-mess.patch
+
+  - name: hxcfloppyemulator
+    buildsystem: simple
+    build-commands:
+      - sed -i /fltk-1.3/d Makefile
+      - sed -i /^FLTKLIB/d ../HxCFloppyEmulator_software/trunk/build/Makefile
+      - sed -i 's@-I ../sources/thirdpartylibs/fltk/fltk-1.3.x/@$(shell fltk-config --cxxflags)@g' ../HxCFloppyEmulator_software/trunk/build/Makefile
+      - sed -i 's@LDFLAGS += -Wl,-rpath=.@LDFLAGS += -Wl,-rpath=. $(shell fltk-config --ldflags)@g' ../HxCFloppyEmulator_software/trunk/build/Makefile
+      - make
```
Comment (barthalion) [inline fr.free.hxc2001.HxCFloppyEmulator.yaml]: You can probably pass `-j$FLATPAK_BUILDER_N_JOBS` here.

**Comment by FakeShemp** [comment]
> It would be nice to contact upstream, but project's page doesn't even load for me.

I've made a tweet mentioning them, which they retweeted, so they're aware at least.
https://twitter.com/Cindertuft/status/1161696036957691905

**Comment by barthalion** [comment]
bot, build fr.free.hxc2001.HxCFloppyEmulator

**Comment by barthalion** [comment]
Alright, it's good enough, I guess.

Repository has been created: https://github.com/flathub/fr.free.hxc2001.HxCFloppyEmulator

Thanks!

**Pull request closed without merging**